src/stores/TransactionStore.js

```diff
@@ -896,21 +967,16 @@ export default class TransactionStore {
   executeWithdraw = async (index, tx) => {
     try {
       const { type, senderAddress, topicAddress } = tx;
-      const contract = this.app.global.qweb3.Contract(topicAddress, getContracts().TopicEvent.abi);
-      const methodName = type === TransactionType.WITHDRAW ? 'withdrawWinnings' : 'withdrawEscrow';
-      const { txid, args: { gasLimit, gasPrice } } = await contract.send(methodName, {
-        methodArgs: [],
-        senderAddress,
-      });
-      Object.assign(tx, { txid, gasLimit, gasPrice });
+      const nbotMethods = window.naka.eth.contract(getContracts().MultipleResultsEvent.abi).at(topicAddress);
+      const txid = await promisify(nbotMethods.withdraw, []);
```

Comment:
think you need to do `await promisify(nbotMethods.withdraw.send, [])`

Reply:
tried `nbotMethods.withdraw`, it succeeded